src/ManagementCenter.md

```diff
@@ -139,7 +139,8 @@ You can use your existing LDAP server for authentication/authorization on Manage
 Provide the details in this form for your LDAP server:
 
 - **URL:** URL of your LDAP server, including schema (`ldap://` or `ldaps://`) and port.
-- **Search base DN:** Base DN to use for searching users/groups.
+- **Distinguished name (DN) of user:** DN of a user that has admin privileges on the LDAP server. Will be used to connect to the server when authenticating users. 
+- **Password:** Password for the same user.- **Search base DN:** Base DN to use for searching users/groups.
```

Comment:
- **Password:** Password for the same user.
- **Search base DN:** Base DN to be used for searching users/groups.